docs/GettingStarted.md

```diff
@@ -7,63 +7,188 @@ permalink: docs/getting-started.html
+Let's get started by writing a test for a hypothetical `sum.js` file:
 
 ```javascript
-// sum.js
-function sum(value1, value2) {
-  return value1 + value2;
+function sum(a, b) {
+  return a + b;
 }
 module.exports = sum;
 ```
 
-We can get up and running with the following 4 steps:
+Create a directory `__tests__/` with a file `sum-test.js`:
 
-1. Create a directory `__tests__/` with a file `sum-test.js`
+```javascript
+jest.unmock('../sum'); // unmock to use the actual implementation of sum
+
+describe('sum', () => {
+  it('adds 1 + 2 to equal 3', () => {
+    const sum = require('../sum');
+    expect(sum(1, 2)).toBe(3);
+  });
+});
+```
+
+Run `npm install --save-dev jest-cli`.
+
+Add the following to your `package.json`:
+
+```js
+"scripts": {
+  "test": "jest"
+}
+```
+
+Run `npm test`:
+
+```
+[PASS] __tests__/sum-test.js (0.010s)
+```
+
+The code for this example is available at
+[here](https://github.com/facebook/jest/tree/master/examples/getting_started).
+
+**And you are ready to enjoy working with Jest!**
+
+### Babel Integration
+
+If you'd like to use [babel 6](http://babeljs.io/), it can easily be enabled:
+
+```
+npm install --save-dev babel-jest
+```
+
+Don't forget to add a [`.babelrc`](https://babeljs.io/docs/usage/babelrc/) file
+in your project's root folder. For example, if you are using ES2015 and
+[React.js](https://facebook.github.io/react/) with the
+[`babel-preset-es2015`](https://babeljs.io/docs/plugins/preset-es2015/) and
+[`babel-preset-react`](https://babeljs.io/docs/plugins/preset-react/) presets:
+
+```js
+{
+  "presets": ["es2015", "react"]
+}
+```
+
+You are now set-up to use all ES2015 features and React specific syntax,
+for example:
 
-  ```javascript
-  // __tests__/sum-test.js
-  jest.dontMock('../sum');
+```js
+jest.unmock('../CheckboxWithLabel');
 
-  describe('sum', function() {
-    it('adds 1 + 2 to equal 3', function() {
-      var sum = require('../sum');
-      expect(sum(1, 2)).toBe(3);
-    });
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+import CheckboxWithLabel from '../CheckboxWithLabel';
+
+describe('CheckboxWithLabel', () => {
+
+  it('changes the text after click', () => {
+    // Render a checkbox with label in the document
+    const checkbox = TestUtils.renderIntoDocument(
+      <CheckboxWithLabel labelOn="On" labelOff="Off" />
+    );
+
+    const checkboxNode = ReactDOM.findDOMNode(checkbox);
+
+    // Verify that it's Off by default
+    expect(checkboxNode.textContent).toEqual('Off');
+
+    // ...
   });
-  ```
+});
+```
 
-2. Run `npm install jest-cli --save-dev`
+Check out the [React tutorial](docs/tutorial-react.html) for more.
 
-    Jest uses ES2015 features and requires a Node.js version of at least 4.0.0
-    to run.
+**And you are good to go!** The next time you run Jest it will print something
+like
 
-3. Add the following to your `package.json`
+ ```
+ Using Jest CLI v<version>, jasmine2, babel-jest
+ ```
```

Comment:
I noticed that if I _already_ have 
```
"scriptPreprocessor": "<rootDir>/node_modules/babel-jest"
```
in my `package.json`, the `babel-jest` part will not be shown here.